h/annotation_html_presenter.py

```diff
@@ -0,0 +1,284 @@
+"""A class that wraps h.api.models.Annotation and adds some HTML properties."""
```

Comment:
I'm not too keen on ridiculously long module names. I'd be inclined to call this `h.presenters`. That way it can turn into a package at some point in the future without having to rename imports all over the codebase.